apps/admin-portal/src/api/application.ts

```diff
@@ -21,11 +21,11 @@ import { GlobalConfig, ServiceResourcesEndpoint } from "../configs";
 import {
     ApplicationBasicInterface,
     ApplicationInterface,
-    ApplicationListInterface,
+    ApplicationListInterface, AuthProtocolMetadataInterface,
     Claim,
     ClaimDialect,
     ExternalClaim,
-    HttpMethods
+    HttpMethods, OIDCdataInterface, OIDCMetadataInterface
 } from "../models";
```

Comment:
Shall we add these models to the core module and import it from there?

Reply:
Since these models are being used only in the scope of Developer Portal, I think adding them to the core module is not necessary.